### PR TITLE
[DOCS] Adjusts note on minimum node size for ELSER and third-party models

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR:
* adjusts the note on the minimum ML node size on the ELSER page to include reference to ESS and autoscaling,
* adds the same note to the third-party models page.

### Preview

* [ELSER requirements](https://stack-docs_2447.docs-preview.app.elstc.co/guide/en/machine-learning/master/ml-nlp-elser.html#elser-req)
* [Third-party models](https://stack-docs_2447.docs-preview.app.elstc.co/guide/en/machine-learning/master/ml-nlp-model-ref.html#ml-nlp-model-ref)

### CODE REVIEW

File: docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc
Comment (droberts195) [inline docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc:41]:
```suggestion
{cloud}/ec-autoscaling.html[deployment autoscaling] turned off. Autoscaling is recommended because it
```

File: docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc
Comment (droberts195) [inline docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc:43]:
```suggestion
required. Better performance can be achieved by using more allocations or more threads per allocation, which will require bigger ML nodes. Autoscaling will provide bigger nodes when required. If not using autoscaling you must provide suitably sized nodes yourself.
```

**Review by droberts195 — COMMENTED** [review]
You don’t have to use my exact suggested words if there’s a better way to say it, but hopefully you get the gist.

Please make the same adjustments to the other file too.

File: docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc
Comment (dimkots) [reply docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc:41]: I got a bit confused here. Don't we recommend 4GB **and** Autoscaling turned **on**?

Comment (dimkots) [reply docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc:41]: Ok, now I got it. You mean "IF you have autoscaling turned OFF, then you should use min 4GB. If you have autoscaling ON, then it takes care of node sizing itself, so don't worry"

**Review by dimkots — APPROVED** [review]
LGTM

File: docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc
Comment (droberts195) [inline docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc:42]:
```suggestion
autoscaling is recommended because it allows your deployment to dynamically 
```

File: docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc
Comment (droberts195) [inline docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc:45]:
```suggestion
Autoscaling provides bigger nodes when required. If autoscaling is turned off, 
```

File: docs/en/stack/ml/nlp/ml-nlp-model-ref.asciidoc
Comment (droberts195) [inline docs/en/stack/ml/nlp/ml-nlp-model-ref.asciidoc:18]:
```suggestion
Autoscaling provides bigger nodes when required. If autoscaling is turned off, 
```

File: docs/en/stack/ml/nlp/ml-nlp-model-ref.asciidoc
Comment (droberts195) [inline docs/en/stack/ml/nlp/ml-nlp-model-ref.asciidoc:15]:
```suggestion
autoscaling is recommended because it allows your deployment to dynamically 
```

**Review by droberts195 — APPROVED** [review]
LGTM if you could just fix a few typos before merging

**Pull request merged**
